patches/chrome-installer-linux-debian-build.sh.patch

```diff
@@ -20,7 +20,15 @@ index 306b59c138b7cb5df21ad6e9d502bb7a46a104c6..f66764e6637e5fdc882ebf4046a702f1
    -O > "${STAGEDIR}/DEBIAN/control"
    rm -f "${DEB_CONTROL}"
  }
-@@ -49,7 +49,7 @@ stage_install_debian() {
+@@ -39,7 +39,6 @@ gen_control() {
+ prep_staging_debian() {
+   prep_staging_common
+   install -m 755 -d "${STAGEDIR}/DEBIAN" \
+-    "${STAGEDIR}/etc/cron.daily" \
```

Comment:
this seems like a lot of patching in different places, can't we just patch the cron script instead to do nothing?